### DISCUSSION TITLE
Unreadable Menu Text with Dark Mode

### DISCUSSION THREAD

**Discussion by sirusbadiee** [burkybang/SuiteAdvanced-Feedback#49] [Q&A] [answered]
Hello there! Super happy with the plug-in, but wanted to see if it was possible if dark mode enabled the text to be a bit more readable. 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/100874186/189376400-a4bdc483-d2dd-4411-88e4-be594073126b.png">

I love the highlighted selection, but the rest of the menu options are pretty hard to read.

Thank you very much!

**Comment by burkybang** [comment, marked as answer]
That is not normal, and it doesn't look like that for me. I wonder if you have another extension that's interfering.

**Comment by sirusbadiee** [comment]
I mean, are you a Nascar driver or something? That was the fastest response
I have ever seen in my life.

Yes, you are 1,000% right - looks like my Chrome was putting a dark mode on
top of the dark mode from SA. Shoulda researched it more before completely
wasting your time.

Mega appreciate the work you do!

On Fri, Sep 9, 2022 at 9:47 AM Adam Smith ***@***.***> wrote:

> That is not normal, and it doesn't look like that for me. I wonder if you
> have another extension that's interfering.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/49#discussioncomment-3609274>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYBTPSQR3GSNGDTAZ564E4TV5NEYLANCNFSM6AAAAAAQIXY3YU>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/49/comments/3609274@
> github.com>
>


-- 
Sirus Badiee
214-235-4751

**Reply by burkybang** [reply]
No worries at all!
